packages/next/server/api-utils/node.ts

```diff
@@ -510,19 +511,23 @@ function setPreviewData<T>(
       httpOnly: true,
       sameSite: process.env.NODE_ENV !== 'development' ? 'none' : 'lax',
       secure: process.env.NODE_ENV !== 'development',
-      path: '/',
       ...(options.maxAge !== undefined
         ? ({ maxAge: options.maxAge } as CookieSerializeOptions)
         : undefined),
+      ...(options.path !== undefined
+        ? ({ path: options.path } as CookieSerializeOptions)
+        : undefined),
```

Comment:
I was also considering doing something like 

```suggestion
      ...(options.path !== undefined
        ? ({ path: options.path || '/' } as CookieSerializeOptions)
        : undefined),
```
but I think it's rather better respec the `cookies` default value

Also, I though about refactoring this `maxAge` and `path` props to have a single object like

```suggestion
      ...({ path: options.path , maxAge: options.maxAge }),
```

but when `path` or `maxAge` is empty, we'd be passing `{ path: undefined, maxAge: undefined }` instead of `{ }`. I rather not pass anything if they're not defined in this function. I think it's more strict